fixtures/__init__.py

```diff
@@ -25,22 +25,9 @@
 Most users will want to look at TestWithFixtures and Fixture, to start with.
 """
 
-# same format as sys.version_info: "A tuple containing the five components of
-# the version number: major, minor, micro, releaselevel, and serial. All
-# values except releaselevel are integers; the release level is 'alpha',
-# 'beta', 'candidate', or 'final'. The version_info value corresponding to the
-# Python version 3.9 is (3, 9, 0, 'final', 0)."  Additionally we use a
-# releaselevel of 'dev' for unreleased under-development code.
-#
-# If the releaselevel is 'alpha' then the major/minor/micro components are not
-# established at this point, and setup.py will use a version of next-$(revno).
-# If the releaselevel is 'final', then the tarball will be major.minor.micro.
-# Otherwise it is major.minor.micro~$(revno).
-from pbr.version import VersionInfo
-
-_version = VersionInfo('fixtures')
-__version__ = _version.semantic_version().version_tuple()
-version = _version.release_string()
+__version__ = (4, 2, 3, "final", 0)
```

Comment:
You can import the generated `fixtures/_version.py` file.

Reply:
Please double check whether I did the right thing.